inject/inject.vala

```diff
@@ -245,6 +245,9 @@ namespace Frida.Inject {
 			var inchan = new IOChannel.unix_new (fd);
 #endif
 			inchan.add_watch (IOCondition.IN, (source, condition) => {
+				if (script_runner == null)
+					return false;
+
```

Comment:
This means we will stop processing `stdin` if input arrives before `start()` is done. We should instead move all of this logic to a method called at the end of `start ()`.